packages/console/src/consts/subscriptions.ts

```diff
@@ -25,15 +25,25 @@ export const tokenAddOnUnitPrice = 80;
 export const hooksAddOnUnitPrice = 2;
 /* === Add-on unit price (in USD) === */
 
+// TODO: Remove this dev feature flag when we have the new Pro202411 plan released.
```

Comment:
Maybe add an owner for this TODO.

Reply:
No need will query the `isDevFeaturesEnabled` flag.